### PR TITLE
correct typos in filter readme

### PR DESCRIPTION
**Pull request by stefanfoulis**

### CODE REVIEW

**Comment by chanwit** [comment]
Please change per my above review and also sign-off with `$ git commit --amend -s` to make this commit pass CI.

**Comment by vieux** [comment]
@stefanfoulis I believe it's the other way around, `node` is valid, but `name` ins invalid

**Comment by vieux** [comment]
@stefanfoulis also, please sign your commit

File: scheduler/filter/README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 963841b138d8        redis:latest        "redis"             Less than a second ago   running                                             node-1      redis8
 ```
 
-As you can see here, the containers were only scheduled on nodes with the redis imagealreayd pulled.
+As you can see here, the containers were only scheduled on nodes with the redis image alreayd pulled.
```
Comment (aluzzardi) [inline scheduler/filter/README.md]: *already

File: scheduler/filter/README.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ For example,
 * `constraint:region!=us*` will match all nodes outside the regions prefixed with `us`.
 * `constraint:name==/node[12]/` will match nodes named `node1` and `node2`.
 * `constraint:name==/node\d/` will match all nodes named with `node` + 1 digit.
-* `constraint:node!=/node-[01]-id/` will match all nodes, except those with ids `node-0-id` and `node-1-id`.
+* `constraint:name!=/node-[01]-id/` will match all nodes, except those with ids `node-0-id` and `node-1-id`.
```
Comment (aluzzardi) [inline scheduler/filter/README.md]: It should be `node`, and we should probably say

> except those with `node-0-id` and `node-1-id` as their IDs or names

**Comment by aluzzardi** [comment]
@stefanfoulis friendly ping. We'd like to get this one in 0.1.0 :)

Thanks

**Comment by stefanfoulis** [comment]
Sorry for the delay... was still finding out about the detail of signing my commits.

**Comment by vieux** [comment]
@stefanfoulis don't worry, I opened a PR here: #289 with some text updated.

**Comment by stefanfoulis** [comment]
closing, since my changes have been incorporated in #289

**Pull request closed without merging**